### PR TITLE
Added queer bricks

### PR DESCRIPTION
**Pull request by halotroop2288**
They are fully functional, but currently look like snowballs when thrown.

### CODE REVIEW

File: src/main/java/io/github/queerbric/queerbrics/QueerbricsMain.java [hunk 1]
```diff
@@ -1,10 +1,19 @@
...
+	
+	// Returns an Identifier of "modid:key" (ex: "queerbrics:rainbow_brick")
+	public static Identifier getId(String key) {
+		return new Identifier(modid, key);
+	}
+	
 	@Override
 	public void onInitialize() {
-
+		QueerbricsItems.registerAll();
```
Comment (jaskarth) [inline src/main/java/io/github/queerbric/queerbrics/QueerbricsMain.java]: would prefer just `register()`

File: src/main/java/io/github/queerbric/queerbrics/registry/QueerbricsItems.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
+package io.github.queerbric.queerbrics.registry;
+
+import io.github.queerbric.queerbrics.QueerbricsMain;
+import io.github.queerbric.queerbrics.datatypes.item.QueerBrickItem;
+import net.minecraft.item.Item;
+import net.minecraft.util.registry.Registry;
+
+public class QueerbricsItems {
+	// Yes I know that this is a silly way to name colours, but it's consistent! - Caroline
+	public static Item // Color Bricks
```
Comment (jaskarth) [inline src/main/java/io/github/queerbric/queerbrics/registry/QueerbricsItems.java:10]: either `camelCase` or `UPPER_SNAKE_CASE` would be preferred imo

**Review by jaskarth — CHANGES_REQUESTED** [review]

File: src/main/java/io/github/queerbric/queerbrics/QueerbricsMain.java (hunk 1 above)
Comment (halotroop2288) [reply src/main/java/io/github/queerbric/queerbrics/QueerbricsMain.java]: IMO not specific enough.
"register" can mean "register a registerable object" or "register an item"
but
"registerAll" clearly means "register all the objects in the class.

Comment (halotroop2288) [reply src/main/java/io/github/queerbric/queerbrics/QueerbricsMain.java]: Changed to init() per conversation on Discord.

**Review by jaskarth — APPROVED** [review]

**Pull request merged**